### PR TITLE
Sitemap: Added MapView + Corrected Mismatching Screenshots

### LINKED ISSUE #492: Sitemaps: Add MapView Sitemap element
(Derived from https://github.com/openhab/openhab-docs/issues/53)

All details needed can be found in this diff: https://github.com/eclipse/smarthome/pull/277/files#diff-10aa8fe84e9119cfe8cac6ea022ff706

The probably most time consuming task here is probably to create a screenshot :)

### PR DESCRIPTION
**Pull request by Confectrician**
Second try. Now with a valid commit history 😄 

Closes #492
Related #532 -> First try with a wrong commit history and some discussion.

Signed-off-by:  Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Hey Jerome, crucial or not, when creating the first screenshots I tried to have all of them at the same size. Any specific reason for breaking with that?

File: configuration/sitemaps.md [hunk 1]
```diff
@@ -447,6 +448,24 @@ List item=<itemname> [label="<labelname>"] [icon="<iconname>"] [separator=""]
 Splits a String Item at each separator into multiple rows.
 -->
 
+### Element Type 'MapView'
+ 
+```perl
+Mapview [item=<itemname>] [label="<labelname>"] [icon="<iconname>"] [height=<heightvalue>]
+```
+
+Displays an OSM map based on a given Location Item.
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Please add a link to OSM

File: configuration/sitemaps.md [hunk 2]
```diff
@@ -447,6 +448,24 @@ List item=<itemname> [label="<labelname>"] [icon="<iconname>"] [separator=""]
 Splits a String Item at each separator into multiple rows.
 -->
 
+### Element Type 'MapView'
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: [Mapview](https://github.com/eclipse/smarthome/pull/277/files#diff-10aa8fe84e9119cfe8cac6ea022ff706)?

I don't think Mapview should be at the end of the list. Priority-wise I'd say you can move it somewhere around Webview

**Comment by Confectrician** [comment]
> Any specific reason for breaking with that?

Nope, i didn't just think 1 second about it and it wasn't mentioned that way in the issue. 😄 
I recognized that those single elements screenshots didn't have page width and made them a bit smaller but big enough to keep everything nice viewable.

_As suggestion for this:
I would add one sentence about the screenshots and some rough dimensions in the comments below the article.
So for the next update one has the chance to recognize the intention._ :)

### tl;dr:
I didn't think about it, but its no problem to change them quickly.

**Comment by Confectrician** [comment]
#### Regarding the #532 discussion of the element order and presentation:
Maybe we could transform the `table of contents` for this page.

I could imagine something like `Element types` which is a internal link to overview table.

I am not that experienced with jekyll toc but is this possible and could this be an alternative to maky this anyway really large article a bit shorter and increase the usability a bit?

##### My opinion:
The Element List makes this TOC very long and the table with its short explanations of every element type transports the information a bit better.


##### Generally:
We can discuss this in another issue of courese, to keep the pr clean and focused on the initial topic.

**Comment by Confectrician** [comment]
Hey @ThomDietrich,

I don't want to annoy you, but is still something to do here?

I would like to grab the nested elements issue too, but don't want to have two branches where the sitemaps.md is changed. 😃 

----

We can discuss about the toc and table sort order another time.
I am fine with the current state and will open another issue, if i think that I've got a better solution.

**Comment by ThomDietrich** [comment]
Sure thing. All looks good. Here we go!

File: configuration/sitemaps.md [hunk 3]
```diff
@@ -347,6 +348,24 @@ Webview url="http://www.openhab.org" height=5
 
 ![Presentation of the Webview element in BasicUI](images/sitemap_demo_webview.png)
 
+### Element Type 'Mapview'
+ 
+```perl
+Mapview [item=<itemname>] [label="<labelname>"] [icon="<iconname>"] [height=<heightvalue>]
```
Comment (mueller-ma) [inline configuration/sitemaps.md]: @Confectrician It seems that label and icon are not displayed in basic ui. Maybe this info should be added somewhere around here.

Comment (mueller-ma) [reply configuration/sitemaps.md]: Docs: https://github.com/openhab/openhab-docs/issues/640
ESH: https://github.com/eclipse/smarthome/issues/5030
